doc/rapidoc.scss

```diff
@@ -61,7 +61,7 @@
 .nav-bar-tag {
   margin-left: $spacing5;
   color: $colorPrimaryForeground;
-  display: none;
+  display: flex;
```

Comment:
That's weird. Any idea why it have worked in all other browsers, but not Safari?

Reply:
css load order in safari is different